subsys/net/ip/net_context.c

```diff
@@ -876,7 +976,8 @@ int net_context_accept(struct net_context *context,
 #endif /* CONFIG_NET_OFFLOAD */
 
 	if ((net_context_get_state(context) != NET_CONTEXT_LISTENING) &&
-	    (net_context_get_type(context) != SOCK_STREAM)) {
+	    (net_context_get_type(context) != SOCK_STREAM &&
```

Comment:
Similar question - can RAW sockets accept() at all?

Reply:
No. There is no client and server role in raw socket to my understanding.

Reply:
So again, this marked as resolved, but isn't resolved. There should be just no this change in the PR.

Reply:
sorry, I missed it.